lib/bowling.rb

```diff
@@ -0,0 +1,39 @@
+class Bowling
```

Comment:
as discussed, worth looking at the problem by looking at the smallest possible element of the problem. So in this case, a roll or maybe a frame. This would then help later on with the development of the code